pkg/cli-runtime/testing/command_table.go

```diff
@@ -223,7 +234,30 @@ func (tc CommandTestCase) Run(t *testing.T, scheme *runtime.Scheme, cmdFactory f
 				t.Errorf("error during prepare: %s", err)
 			}
 		}
+		if len(tc.GivenObjects) != 0 {
```

Comment:
remove the if condition
```
var pods []client.Object
for loop {
}
if len(pods) > 0 {
   fake builder
}
```

Reply:
We would need a default builder as the vendor k8s file is throwing null pointer exception if we do not pass a builder instance when calling unstructured function.